src/50_Samples_%26_Templates/Product.html

```diff
@@ -278,53 +346,177 @@ <h4 class="category"><a href="/samples_templates/product_listing/preview/">More<
+  <!-- We use the `amp-state` component (part of [amp-bind](https://www.ampproject.org/docs/reference/components/amp-bind))
+      to configure the product price depending on the color and the size. We also configure the `defaultSize`
+      for each product when you switch between colours -->
+  <amp-state id="product">
+    <script type="application/json">
+    {
+       "green":{
+          "sizes":{
+             "S":"$5.99",
+             "M":"$5.99",
+             "L":"unavailable"
+          },
+          "defaultSize":"S"
+       },
+       "golden":{
+          "sizes":{
+             "S":"$9.99",
+             "M":"unavailable",
+             "L":"$9.99"
+          },
+          "defaultSize":"S"
+       },
+       "red":{
+          "sizes":{
+             "S":"$7.99",
+             "M":"$7.99",
+             "L":"$7.99"
+          },
+          "defaultSize":"S"
+       }
+     }
+  </script>
+</amp-state>
 
-    <h5 class="price-description">Price: $1.99</h5>
     <!-- #### Product Configuration -->
-    <!-- The add to cart action is implemented using  [amp-form](/components/amp-form/). In our sample we use the [amp-selector](/components/amp-selector/) for selecting different product properties. Pressing the `ADD TO CART` button adds the product to a shopping cart page using the properties you have selected. Notice that the button URL contains the query `clientId={{ClientId}}`. If an item is successfully added to the shopping cart, we redirect the user to the shopping cart page. The redirect target is configured via a header in the server's form response ([redirecting after submission](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md#redirecting-after-a-submission)):
+    <!-- We are using [amp-bind](https://www.ampproject.org/docs/reference/components/amp-bind) to update the price of the product
+      depending on the selected colour. -->
+    <!--We bind to the `text` attribute: the value of text is the
+      value of expression `product[selectedColor].price`. When this expression is evaluated,
+      it gets the price value from the json inside the `amp-state`.-->
+    <!--Notice how we set a default value for the price: this is because the value defined inside `amp-state`
+      will not be accessed at the page load. We set `green` and `S` as default color and default size
+      as the correspondent variables have null value at default.-->
+    <!--As soon as the state of this variable changes, the expression is evaluated.
+      See below to understand how we change the state of the 2 variables `selectedColor` and `selectedSize`. -->
+      <p class="price-description">Price: <span [text]="product[selectedColor || 'green'].sizes[selectedSize || 'S']">$1.99</span>
+    </p>
+    <!-- The add to cart action is implemented using  [amp-form](/components/amp-form/). In our sample we use the [amp-selector](/components/amp-selector/) for selecting different product properties.
```

Comment:
This feels like a different topic - it might make sense to start a new paragraph here.